required/tools/varioref.dtx

```diff
@@ -169,7 +169,7 @@
 % comes out as ``\ldots~see the example which shows~\ldots'', which
 % could be misleading.
 %
-% A space in front of |\vpageref| it will be ignored if
+% A space in front of |\vpageref| will be ignored if
 % the command doesn't produce any text at all.
```

Comment:
Why? The original sentence looks right to me.

"A space (in from of ...) will be ignored (if ...)."